Dockerfile.php7

```diff
@@ -0,0 +1,7 @@
+FROM php:7.0
```

Comment:
Are you saying I can just overwrite the default Dockerfile and Dockerfile.MySQL with mine?